CMakeLists.txt

```diff
@@ -486,10 +486,15 @@ if (NETWORKIT_BUILD_TESTS)
 					networkit_state
 			)
 		endif()
-
+		
+		if(WIN32)
+			set(NETWORKIT_TESTS_CXX_FLAGS "${NETWORKIT_CXX_FLAGS} /wd4996")
```

Comment:
Only minor, but no need to introduce a new variable. Just reuse `NETWORKIT_CXX_FLAGS`. We do so in other parts of the CMake-file.